cscs-checks/apps/tensorflow/tf_horovod_check.py

```diff
@@ -0,0 +1,54 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class TensorFlowHorovodTest(rfm.RunOnlyRegressionTest):
+    def __init__(self):
+        super().__init__()
+        self.descr = 'Distributed training with TensorFlow and Horovod'
+        self.valid_systems = ['daint:gpu', 'dom:gpu']
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        tfshortver = '1.11'
+        self.sourcesdir = 'https://github.com/tensorflow/benchmarks'
+        self.modules = ['Horovod/0.15.0-CrayGNU-18.08-tf-%s.0' % tfshortver]
+        self.reference = {
+            'dom:gpu': {
+                'img_sec': (1133.6, None, 0.05),
```

Comment:
What is `img_sec`? Please add also units in the tuple.  Keep in mind that `sec` is not an SI unit. It should be written as `s`.